test/cri-containerd/main_test.go

```diff
@@ -98,7 +91,6 @@ var (
 	flagVirtstack             = flag.String("virtstack", "", "Virtstack to use for hypervisor isolated containers")
 	flagVMServiceBinary       = flag.String("vmservice-binary", "", "Path to a binary implementing the vmservice ttrpc service")
 	flagContainerdServiceName = flag.String("containerd-service-name", "containerd", "Name of the containerd Windows service")
-	flagSevSnp                = flag.Bool("sev-snp", false, "Indicates that the tests are running on hardware with SEV-SNP support")
 )
 
 // Features
```

Comment:
we probably should also remove `featureLCOW*` and `featureCRIPlugin` (and tests that use the latter as well: containerdrestart_test.go, helper_cri_plugins_test.go, and co), since upstream doesn't implement the plugin service